modules/api/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -17,16 +17,33 @@
 
 package org.apache.ignite.lang;
 
+import static java.util.regex.Pattern.DOTALL;
+
 import java.util.HashMap;
+import java.util.HashSet;
 import java.util.Locale;
 import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import org.apache.ignite.error.code.annotations.ErrorCodeGroup;
+import org.jetbrains.annotations.Nullable;
 
 /**
  * Defines error groups and its errors.
  */
 @SuppressWarnings("PublicInnerClass")
 public class ErrorGroups {
+    /** Additional prefix that is used in a human-readable format of ignite errors. */
+    public static final String IGNITE_ERR_PREFIX = "IGN";
```

Comment:
Perhaps, the following code should be updated as well https://github.com/apache/ignite-3/blob/6b4a633dcce8360741da90f6dc42a73f574defca/modules/platforms/dotnet/Apache.Ignite/ErrorGroups.cs#L39

Reply:
done